assets/js/main.js

```diff
@@ -2,12 +2,17 @@ $("body").on({
   mouseenter: function() {
     var submenu = $(this).find(".sub-menu");
 
+
```

Comment:
are this extra spaces on purpose?

Reply:
I am applying this fix.

Reply:
No, no lo voy a corregir porque no entiendo del todo como funciona esto, mejor que @damianmuti le de una mirada el Jueves.